index.html

```diff
@@ -54,5 +55,148 @@ <h1>I’m Fadahunsi Glad to see you!</h1>
             </ul>
         </div>
     </section>
+    <main>
+        <section id="myWork" class="my-works">
```

Comment:
- [ ] please try to use the right background color for your section so you shoud use the one provided in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/170544455-8a93746b-2b54-4a21-9aad-530eb9649619.png)